app/controllers/application_controller.rb

```diff
@@ -11,8 +11,7 @@ class ApplicationController < ActionController::Base
   include Vmdb::Logging
 
   if Vmdb::Application.config.action_controller.allow_forgery_protection
-    db = MiqDatabase.connected? ? MiqDatabase.first : nil
-    protect_from_forgery :secret => db ? db.csrf_secret_token : SecureRandom.hex(64), :except => :csp_report, :with => :exception
+    protect_from_forgery :secret => SecureRandom.hex(64), :except => :csp_report, :with => :exception
```

Comment:
The fact this isn't the "real" secret (in any process a reader is likely to care about) seems worth a comment. Otherwise I can easy foresee some not-fun debugging confusion in someone's future.